sdk/src/types/block/address/multi.rs

```diff
@@ -121,6 +122,17 @@ impl MultiAddress {
     pub fn threshold(&self) -> u16 {
         self.threshold
     }
+
+    /// Hash the [`MultiAddress`] with BLAKE2b-256.
+    #[inline(always)]
+    pub fn hash(&self) -> [u8; 32] {
```

Comment:
why `pub`? users gonna need to work with the hash of it?

Reply:
I asked myself the same but since it's public anyway through the display I think it's fair to have the method pub

Reply:
yea, figured it wouldnt hurt to have available